ui/dashboards/src/components/Dashboard/Dashboard.stories.tsx

```diff
@@ -168,7 +168,7 @@ function formatProviderParameters(dashboardState: DashboardResource) {
         initialVariableDefinitions: dashboardState.spec.variables,
       },
     },
-    WithTimeRange: {
+    withTimeRange: {
```

Comment:
I really wish the storybook typing worked better for `parameters`, so I would've caught this typo previously, although it wasn't why the chart was having issues. Just fixing it while I'm in here.